src/server/router.ts

```diff
@@ -118,11 +118,21 @@ export class MockerRouter implements IMockerRouter {
   }
 
   /**
-   * Create a new router with the given baseURL,
-   * relative `baseURL` will be resolved to current origin
+   * Create a new router with the given path,
+   * the path will be resolved against current baseURL.
    */
-  base(baseURL: string = this.baseURL): MockerRouter {
-    const url = new URL(baseURL, this._origin);
+  base(path: string = this.baseURL): MockerRouter {
```

Comment:
I think it's unnecessary because that will not cause chaos. Also URL parsing is gonna be complex.